src/components/Courses/components/CourseCard/CourseCard.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './CourseCard.css';
+
+const CourseCard = ({ course, authors }) => {
```

Comment:
You are not passing authors anywhere the value of authors is undefined please review this part